src/ape_accounts/accounts.py

```diff
@@ -131,11 +132,19 @@ def sign_message(self, msg: SignableMessage) -> Optional[MessageSignature]:
             return None
 
         signed_msg = EthAccount.sign_message(msg, self.__key)
-        return MessageSignature(v=signed_msg.v, r=signed_msg.r, s=signed_msg.s)  # type: ignore
+        return MessageSignature(  # type: ignore
+            v=signed_msg.v,
```

Comment:
Kind of weird, I just noticed we do the opposite of this in `ape.types.signatures`

Reply:
I think what you see is that `v` is converted to `bytes`

this logic is somewhat messed up, yeah